tests/test_minimal/test_tools/s3_tools.py

```diff
@@ -46,3 +47,37 @@ def test_estimate_total_conversion_runtime():
         1235294.1176470588,
         12352941.176470589,
     ]
+
+
+def test_submit_aws_batch_job():
+    job_name = "test_submit_aws_batch_job"
+    docker_image = "ubuntu:latest"
+    command = "echo 'Testing NeuroConv AWS Batch submission."
+
+    submit_aws_batch_job(
+        job_name=job_name,
+        docker_image=docker_image,
+        command=command,
+    )
```

Comment:
What this is currently testing is simply that the calls via `boto3` (including the final submission) do not fail (which would have raised errors)

They do not guarantee the job itself succeeded, since that is separate from the submission

In a follow-up, I plan to leverage the helper function for table status updates to actually get these minimal tests to flag their completion at runtime, and then add an assertion (likely with some delay or repeat w/ timeout) that the status table was updated to 'completed' by the end of it all

Reply:
OK.